torch/_inductor/overrides.py

```diff
@@ -199,6 +227,26 @@ def fuse_conv_binary_eval(conv: nn.Module, binary_op_name: str):
     )
 
 
+def is_bfloat16_module(m):
+    weight_is_bf16 = m.weight.dtype == torch.bfloat16
+    bias_is_bf16 = m.bias is None or m.bias.dtype == torch.bfloat16
+    return weight_is_bf16 and bias_is_bf16
+
+
+def bf16_only_node(m):
+    if type(m) in [nn.Linear]:
+        return True
+    else:
+        return False
+
+
+def fuse_linear_unary_eval(linear: nn.Module, unary: nn.Module):
```

Comment:
Since it returns a fused module, not doing fusion, so I guess naming it `fused_linear_unary_eval` is more appropriate?

Reply:
Changed.